### DISCUSSION TITLE
1/2/3 diode for 1 key

### DISCUSSION THREAD

**Discussion by Kir2406** [onlaj/Piano-LED-Visualizer#297] [Q&A]
How to set the number of diodes for 1 key. Sometimes 1 diode works, and sometimes 3. Two diodes for 1 key turned on once.
How do I set it up?

Ideally, some kind of instruction would be found ...

**Comment by onlaj** [comment]
That option is called "Sides Color". If you turn it on the adjacent leds will light up after pressing the key. You can also set a custom color for those additional leds.

**Reply by Kir2406** [reply]
Yes i did it.
So the 2 diodes are not working? Only 1 or 3?

**Reply by onlaj** [reply]
Yep
Maybe in future I will add support for 2 leds.

**Reply by Kir2406** [reply]
Ok! 
Thank you so much!
